CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## [v0.23.1](https://github.com/mongodb/go-client-mongodb-atlas/tree/v0.23.1) (March 01, 2023)
+
+[Full Changelog](https://github.com/mongodb/go-client-mongodb-atlas/compare/v0.23.0...v0.23.1)
+
+**Breaking changes:**
+
+- fix: make org owner optional [\#393](https://github.com/mongodb/go-client-mongodb-atlas/pull/393) ([gssbzn](https://github.com/gssbzn))
```

Comment:
Rule of thumb for relase notes that every breaking change has migration path. 
Since this is generated it will be hard to do it. Maybe worth adding those on github release notes?

Reply:
I also autogenerate the GH ones but I can give it a try